test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/ConditionalCreateTests.cs

```diff
@@ -140,5 +140,20 @@ public async Task GivenAResource_WhenCreatingConditionallyWithEmptyIfNoneHeader_
             Assert.Single(exception.OperationOutcome.Issue);
             Assert.True(exception.Response.Resource.Issue[0].Diagnostics.Equals(string.Format(Core.Resources.ConditionalOperationNotSelectiveEnough, "Observation")));
         }
+
+        [Fact]
+        [Trait(Traits.Priority, Priority.One)]
+        public async Task GivenAResource_WhenCreatingConditionallyANewDuplicatedSearchParameterResource_TheServerShouldFail()
+        {
+            var id = Guid.NewGuid();
+            var resourceToCreate = Samples.GetJsonSample<SearchParameter>("SearchParameterDuplicated");
+            resourceToCreate.Id = id.ToString();
+
+            using FhirClientException ex = await Assert.ThrowsAsync<FhirClientException>(() => _client.CreateAsync(
+                resourceToCreate,
+                $"SearchParameter/id={id}"));
+
```

Comment:
Note: How can I make sure I'm reaching the ConditionalCreate function? 🤔

Reply:
Make sure to add the header to the request, If-None-Exist

Reply:
Given the other tests in the same file, looks like we do not have a ConditionalCreate function, but I tested manually that I'm calling the **ConditionalCreate** function and not the normal **Create**.